containers/datalab/config/ipython.py

```diff
@@ -28,3 +28,7 @@
 
 # Enable matplotlib renderings to show up inline in the notebook.
 c.InteractiveShellApp.matplotlib = 'inline'
+
+# Customize the Save as HTML handling
+c.TemplateExporter.template_path = ['/datalab/nbconvert']
+c.TemplateExporter.template_file = 'as_html'
```

Comment:
Also to confirm, is there a reason ipython.config needs this? I would have expected the kernel to know nothing about notebooks... maybe just config/nbconvert.py suffices?